glamorous/button.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import glamorous from 'glamorous';
+
+const MyStyledButton = glamorous.button({
+  backgroundColor: '#ff0000',
+  width: 320,
+  padding: 20,
+  borderRadius: 5,
+  border: 'none',
+  outline: 'none',
+  ':hover': {
+    color: '#fff',
+  },
+  ':active': {
+    position: 'relative',
+    top: 2,
+  },
+  '@media(max-width: 480px)': {
+    width: 160,
+  },
+});
+
+const Button = () => (
+  <glamorous.Div textAlign="center">
```

Comment:
I love that this is a great demonstration of both the factory API and the component API.

Some people don't like the `<glamorous.Div />` stuff and would rather see:

```js
const {Div} = glamorous
<Div />
```

While others prefer it as you have it. Any chance we could show both with a comment that you can do either?